Unit 3.1: npm and yarn scripts

frankiebarrios opened the pull request:
## What It Does
This pull request is set to complete Unit 3.1 of the ux-onboarding-project.

This work is taking place to complete unit 3.1 of the ux-onboarding-project.

**Implementation:**
- Added script to package.json to run server at http://localhost:1820 using 'yarn start'.
- Updated README with further instructions.

[closes #15 ](https://github.com/frankielb/frankie-barrios-ux-aop/issues/12)

## How To Test
- Install dependencies using instruction in README.
- Start server using yarn start.
- Access site at http://localhost:1820.

## Notes/Caveats

None
## Checklist

Under penalty of public shaming, I avow that I:
**Many items in list N/A** 
- [X] Reviewed the diff to look for typos, whitespace errors, and console.log() statements
- [X] Verified that there are no compiler or linter errors or warnings
- [X] Verified the build in production(optimized) mode
- [X] Updated the docs, if necessary
- [X] Included the appropriate labels
- [X] Referenced applicable issues (i.e. Closes #XXXX, Fixes #XXXX)

Browsers tested:
**Many items in list N/A** 
- [X] Chrome
- [X] Firefox
- [X] Safari
- [X] Edge
- [X] Internet Explorer

## Dependencies
1. Install Node and npm
2. Install Yarn

Commit:
Installed http-server and created scripts.

frankiebarrios committed:
Updated README

Commit:
Completed Unit 3.1

Commit:
Merge branch 'Unit-3.1-npm-scripts' of github.com:frankielb/frankie-barrios-ux-iop into Unit-3.1-npm-scripts

kddrake commented on README.md:
````diff
@@ -1,6 +1,14 @@
-
+Run npm install to install dependencies
+```
+$ npm install 
````
Should be 
```suggestion
$ yarn install 
```

Using both `npm` and `yarn` can cause problems, as they use slightly different lock file. You'll want to make sure your consistently using `yarn`

frankiebarrios replied:
Got it

kddrake reviewed (changes requested):
Can you get `master` merged into this branch so that all your additions you made in unit 2 aren't also being committed, again, here?

Commit:
Merge branch 'master' of github.com:frankielb/frankie-barrios-ux-iop into Unit-3.1-npm-scripts

frankiebarrios committed:
Update README.md

Using both npm and yarn can cause problems, as they use slightly different lock file. You'll want to make sure your consistently using yarn

Co-Authored-By: Kean Drake <kean.drake@banno.com>

kddrake reviewed (changes requested):
Derp, forgot to say you should remove `package-lock.json`, run `yarn install`, an commit the `yarn.lock` file

Commit:
Removed previous package-lock.json and ran yarn install.

kddrake commented on package.json:
```diff
@@ -2,9 +2,11 @@
+  "main": "index.html",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
```
now that we have an actual script to run, lets remove this

kddrake commented on package.json:
```diff
@@ -2,9 +2,11 @@
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "say-hello": "echo 'Hello World'",
```
now that we have an actual script to run, lets remove this, too

Commit:
Removing test scripts.

kddrake reviewed (changes requested):
I'm unable to actually hit `http://localhost:1820`, but was able to hit the provided urls given by `http-server`. May want to ensure we can use `https`... I'm guessing that's why I can't use localhost

Commit:
Added default address and opening command to script.

Commit:
Revert "Added default address and opening command to script."

This reverts commit 478ed6f717d0fe908bcd988ceffed4ffa5ec83cb.
